main.py

```diff
@@ -95,26 +97,14 @@ async def get_experiment_info(file: str) -> Any:
     return remote.examine(file)
 
 
-@app.get("/experiment/code/")
-async def get_experiment_code(file: str) -> str:
-    """Gets code of the given experiment file and returns it.
-    
-    Args:
-        file: The path of the experiment file.
-    """
-    full_path = posixpath.join(configs["master_path"], configs["repository_path"], file)
-    with open(full_path, encoding="utf-8") as experiment_file:
-        code = experiment_file.read()
-    return code
-
-
 @app.get("/experiment/submit/")
-async def submit_experiment(
+async def submit_experiment(  # pylint: disable=too-many-arguments
     file: str,
     args: str = "{}",
     pipeline: str = "main",
     priority: int = 0,
-    timed: Optional[str] = None
+    timed: Optional[str] = None,
+    visualize: Optional[bool] = False
```

Comment:
Why should it be `Optional`?

Reply:
Umm.. it is just for convenience! I intended the default options is `False`.

Reply:
`Optional[bool]` is `bool` or `None`. If you don't need `None`, you should set it as `bool`.

Reply:
😮 I was completely deluded! I will remove it.